language/use-cases/document-qa/question_answering_documents_langchain_bigquery_vector_search.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<p>You're only loading one document, do you need to use the DirectoryLoader ? There should be another loader for single files</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Changed it to `GCSFileLoader`